azure/script.sh

```diff
@@ -0,0 +1,15 @@
+export RESOURCE_GROUP_NAME=onurctirtir_test_automation
```

Comment:
Dont forget to remove this file before merging :)

Reply:
yep, it is removed and force-pushed just after